### PR TITLE
fix: Better element ID naming for disambiguation (fixes #74)

### LINKED ISSUE #74: Improve readability for 'simple-template' example
My issue relate to: https://github.com/mdn/web-components-examples/tree/712ce27cd5b3c894ebc9b9958ec0973e3e86157b/simple-template

My point is the custom-element name and the template id name is the same that confused for anyone start to read from `main.js` file. I would like to replace `<template id="my-paragraph">` to `<template id="template-for-my-paragraph">`.

If you guy think it is a right way, I would like to make my PR to this repo.

### PR DESCRIPTION
**Pull request by ngdangtu-vn**
#### Summary
Suggest a new name for template ID in **simple-template** example.

#### Related issues
Fixes #74

#### Related content pull request
Later...

### CODE REVIEW

**Comment by ngdangtu-vn** [comment]
@bsmth I would like a feedback about the name. I try to go as simple as possible. It should change to much but still distinguishable.

**Comment by bsmth** [comment]
Hi there, sorry I forgot to reply. I've added myself as a reviewer so I don't forget.

File: simple-template/index.html [hunk 1]
```diff
@@ -8,15 +8,17 @@
 <body>
   <h1>Simple template</h1>
 
-  <template id="my-paragraph">
+  <template id="custom-my-paragraph">
```
Comment (bsmth) [inline simple-template/index.html:11]: After thinking about this, I came back to `custom-p-element` again (what I thought of first in the linked issue) what do you think?

```suggestion
  <template id="custom-p-element">
```

File: simple-template/index.html [hunk 2]
```diff
@@ -31,4 +33,4 @@ <h1>Simple template</h1>
   </my-paragraph>
 
 </body>
-</html>
+</html>
```
Comment (bsmth) [inline simple-template/index.html:36]: (minor one, it's flagged here as no newline at end of file)

File: simple-template/main.js [hunk 3]
```diff
@@ -3,10 +3,10 @@ customElements.define('my-paragraph',
     constructor() {
       super();
 
-      const template = document.getElementById('my-paragraph');
+      const template = document.getElementById('custom-my-paragraph');
```
Comment (bsmth) [inline simple-template/main.js:6]: Same as comment above:

```suggestion
      const template = document.getElementById('custom-p-element');
```

**Comment by bsmth** [comment]
Hi @ngdangtu-vn what do you think about these changes? 😄 Some comments for you to have a look at. Thanks!

**Comment by ngdangtu-vn** [comment]
I'm so sorry, I thought I left feedback already. Turn out I didn't :))

I personally don't mind to use shorter name. However, because this example will be used across the MDN doc as small pieces (not a whole example), I decided to go with more semantic than short words.

What do you think?

**Comment by bsmth** [comment]
> I'm so sorry, I thought I left feedback already. Turn out I didn't :))

Not a problem!

> I personally don't mind to use shorter name. However, because this example will be used across the MDN doc as small pieces (not a whole example), I decided to go with more semantic than short words.
> 
> What do you think?

Yes, I see what you mean. What about `custom-paragraph` (to get rid of `my` in this case)?

**Comment by ngdangtu-vn** [comment]
> What about custom-paragraph (to get rid of my in this case)?

Yup, that's better.

File: simple-template/index.html (hunk 2 above)
Comment (bsmth) [inline simple-template/index.html:36]:
```suggestion
</html>

```

**Comment by bsmth** [comment]
> > What about custom-paragraph (to get rid of my in this case)?
> 
> Yup, that's better.

Nice, I've edited the suggestions if you want to have a look 👀

**Review by bsmth — APPROVED** [review]
Thanks a lot! This looks good to me, I think we can merge and create a PR in mdn/content to update the code and prose there to reflect the naming change!

**Comment by bsmth** [comment]
Actually I'll hold off on merging until we're happy with the content PR. The pages that need an update are in the linked issue, as far as I can see.

**Comment by ngdangtu-vn** [comment]
@bsmth So for now, we agree with the new name for the template tag ID and starting to update the content on other MDN pages. Could you confirm if I understand correctly? If so, I'll start on update the tag ID on other MDN pages :)

**Comment by bsmth** [comment]
> @bsmth So for now, we agree with the new name for the template tag ID and starting to update the content on other MDN pages. Could you confirm if I understand correctly? If so, I'll start on update the tag ID on other MDN pages :)

Yes! That sounds good, when you are ready with changes, you can tag me in the PR and I will happily review! Thank you

**Comment by bsmth** [comment]
Let's get it merged 🚢

**Pull request merged**